### DISCUSSION TITLE
这款工具帮助到你了么~ | Leave your comments if this tool helped you~

### DISCUSSION THREAD

**Discussion by ludoux** [ludoux/qbittorrent-torrents-exporter#1] [通用 | General]
倘若这款工具帮助到了你，希望您可以留下评论并提供如下信息，帮助开发者更好的完善！
Please leave a comment with the information below, which will make this tool more perfect!
```
qbittorrent-torrents-exporter version 版本: (当下最新的/latest)
qBittorrent version 版本: 
qBittorrent tasks amount 合计任务数: 
```

**Comment by BackMountainDevil** [comment]
wow,我不会是第一个留言的吧，准备测试
- pwd: `~/.local/share/qBittorrent/` (感觉这个要写在 README 中)
- cmd: `./qbittorrent-torrents-exporter -qh "http://localhost:8080" -qu "admin" -qp "中国靓仔加密密码" -githubchannel` （从issue 里抄的，还是建议写入 readme）

运行不需要安装 go, nice （建议写入 readme），默认导出位置～/.local/share/qBittorrent/export/

```
qBittorrent version 版本: github.com/ludoux/qbittorrent-torrents-exporter v0.2.1 linux amd64
qBittorrent tasks amount 合计任务数: 162
```

**Reply by BackMountainDevil** [reply]
又是我，qBittorrent v5.0.2 (64-bit) 上直接导出种子文件时，若存在同名文件，则导出会失败。因为在不同的 pt 站点做种，因此会出现这个问题，用这个则不会遇到这个问题。
